pills/04/relative-path.txt

```diff
@@ -1,2 +1,3 @@
-nix-repl> 2/3
+nix-repl> 6/3
```

Comment:
I think you meant `2/3`. Can you revert all changes to this file including the trailing newline at the end of this file?

Reply:
I actually meant 6/3 here, and on the next line. Since it's the first time the user is seeing division in Nix, I wanted to use a simple example where the denominator evenly divides the numerator. But I am removing the trailing newline.